src/app.js

```diff
@@ -71,7 +73,39 @@ function listen (koaApp, config, core, backend, routeBuilder, routeBroadcaster)
   }).catch((err) => log.error(err))
 }
 
-function createApp (config, core, backend, routeBuilder, routeBroadcaster, routingTables, infoCache, balanceCache) {
+function addPlugin (config, core, backend, routeBroadcaster, tradingPairs, id, options, tradesTo, tradesFrom) {
+  return co(function * () {
+    core.addClient(id, new ilpCore.Client(Object.assign({}, options.options, {
+      connector: config.server.base_uri,
+      _plugin: require(options.plugin),
+      _log: log.create(options.plugin)
+    })))
+
+    yield core.getClient(id).connect()
+
+    if (tradesTo) {
+      tradingPairs.addPairs(tradesTo.map((e) => [options.currency + '@' + id, e]))
+    } if (tradesFrom) {
```

Comment:
I would put the new `if` on a separate line to make it clearer that they're separate. Right now it's easy to confuse this with an `else if` due to the unusual formatting.